physics/Interstitials/UFS_SCM_NEPTUNE/GFS_suite_interstitial_3.F90

```diff
@@ -102,7 +102,26 @@ subroutine GFS_suite_interstitial_3_run (otsptflag,                 &
          enddo
       endif
 
-
+      ! In case of using prognostic updraft velocity, initialize updraft velocity here
+      ! since progomega_calc is called from both deep and shallow schemes.
```

Comment:
Since sigmaout and omegaout are intent(out) in the deep and shallow convection schemes, this initialization is kinda "forgotten" as soon as those schemes are entered. Intent(out) variables become undefined when the subroutine is entered. If you want to initialize like this, I think that you may need to change sigmaout and omegaout to intent(inout) in samf[deep/shal]cnv. If you do this, you can disregard the comments about making sure that they're initialized within the scheme.